content/cpp/concepts/variables/terms/local-variables/local-variables.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'Local Variables'
+Description: 'A variable defined inside a function body between braces is called a local variable.'
+Subjects:
+  - 'Game Development'
+  - 'Computer Science'
+Tags:
+  - 'Variables'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Local Variable is a variable in which it's scope is only limited to the function where it is defined. It exists and can only be accessed inside a function.
+
+## Syntax
+
+```pseudo
+void local_variable(){
+    int a_local_variable;
+}
+```
+
+The variable **`a_local_variable`** is local to the **`local_variable`** function it is defined in. It can only be used inside that function and the life of the variable ends when the function exits. 
```

Comment:
```suggestion
The variable `a_local_variable` is local to the `local_variable` function in which it is defined. It can only be used inside that function and the life of the variable ends when the function exits.
```